### DISCUSSION TITLE
How to export author of sync

### DISCUSSION THREAD

**Discussion by scibol** [argoproj/argo-cd#7727] [Q&A] [answered]
# Summary

I would like to export "who did what" in ArgoCD, e.g. User X synced application Y (or sub-application) at time Z

# Motivation

As audit trail, and to create metrics

# Proposal

Logs would be the best option

**Comment by jessesuen** [comment, marked as answer]
> Logs would be the best option

We emit normal kubernetes events about this. We feel this is sufficient enough audit trails since long-term storage of these type of events would be out of scope for Argo CD. Also, I believe these events also already appear in logs as well.
